### DISCUSSION TITLE
Custom title bar colour for Mac OS

### DISCUSSION THREAD

**Discussion by p4-k4** [neovide/neovide#1311] [Ideas]
![28492796-2c6a02e6-6f0a-11e7-917e-aec4cd3fc4a3](https://user-images.githubusercontent.com/61640764/168423730-746eec6f-5e17-4eec-b592-c80913ecb4d2.png)

Matched titlebar colour with main background colour gives a super clean look.
Additionally, removal of the yucky Mac OS window bevel completes the look.

**Comment by LoipesMas** [comment]
I think this was already mentioned in #1298
